### DISCUSSION TITLE
PrimeNG Ultima --> How to override existing layout styles,  layout variables, theme styles,  theme variables of PrimeNG Ultima template

### DISCUSSION THREAD

**Discussion by nazrulcsebd** [primefaces/community#1888] [PrimeNG Templates]
Dear PrimeNG team,
How to override existing layout styles,  layout variables, theme styles, and theme variables of PrimeNG Ultima template. In other theme (like Freya, California, Omega) there is a folder called **overrides** and we override the necessary style and variables there. In Ultima template, there is no override folder. Is there any way to override the theme like other template? Please give me a guide on how to do that.

**Comment by alpesh88ww** [comment]
I want to do this also.. lets see when they give Proper Ans.

**Reply by SiebeDom** [reply]
Hello,
I think I asked the same question here (with our current fix):
https://github.com/orgs/primefaces/discussions/288
It might be good if overrides come available again by default.
